tantalus/models.py

```diff
@@ -978,4 +980,111 @@ def get_absolute_url(self):
         return reverse("submission-detail", args=(self.id,))
 
     def __str__(self):
-        return str(self.sample.sample_id)
\ No newline at end of file
+        return str(self.sample.sample_id)
+
+
+class Curation(models.Model):
+    """
+    Simple text curation associated with datasets.
+    """
+
+    history = HistoricalRecords()
+
+    name = create_id_field(unique=True)
+
+    owner = models.ForeignKey(
+        account.models.User,
+        on_delete=models.SET_NULL,
+        null=True,
+    )
+    sequencedatasets = models.ManyToManyField(
```

Comment:
consider using a through model here, that would allow you to track history of the curation dataset membership using django-simple-history